### DISCUSSION TITLE
How to take my project in github repository

### DISCUSSION THREAD

**Discussion by 24sa12** [github/india#851] [GitHub Platform]
I have made a project on azure now should I take this project to repository so that I can use that repository URL in submitting the project on future ready talent. 
Please can  u help me as I don't know anything about github, I'm new in github...
Please help me

**Comment by FRTMentor** [comment]
GitHub repository is very user friendly easy to learn and understood.
Please visit the GitHub site, try to create an account similar to the FRT.
And as GitHub landing page will guide to create and and ad the repository.
Also please go through some GitHub learning video's or google links before creating.
